packages/x-components/src/x-modules/facets/components/facets/__tests__/facets-provider.spec.ts

```diff
@@ -122,13 +122,13 @@ describe('testing Facets component', () => {
     emittedEvents = wrapper.emitted('UserChangedSelectedFilters');
     expect(emittedEvents).toHaveLength(2);
     emittedFilters = emittedEvents?.[1][0];
-    expect(areFiltersDifferent(emittedFilters, [])).toBe(false);
+    expect(areFiltersDifferent(emittedFilters, [redFilter, bigFilter])).toBe(false);
```

Comment:
The doubt I have:

What does it happen when we set the props again in the provider?

Reply:
Right now we are mutating the provided filters, which I think we shouldn't.

To solve it we have to clone the facets and filters before provide them

Reply:
I found the problem:
The test is mutating the provided filters:
<img width="638" alt="image" src="https://user-images.githubusercontent.com/5759712/181459641-a2df13d4-6637-4909-928f-d5694fb74bb4.png">
Instead of mutating the state filters:
<img width="638" alt="image" src="https://user-images.githubusercontent.com/5759712/181459713-5a2c4842-fb47-4426-afe3-6b3070425bda.png">

Reply:
Ok, so, changed and documented the mutate state filters in the tests.
Changed the `JSON.parse(JSON.stringify())` by `clone`